DevEdu.API/DevEdu.Db/StoredProcedures/User/User_SelectByEmail.sql

```diff
@@ -3,24 +3,24 @@
 AS
 BEGIN
 	SELECT 
-		u.Id,
-		u.FirstName,
-		u.LastName,
-		u.Patronymic,
-		u.Email,
-		u.Password,
-		u.Username,
-		u.IsDeleted,
-		u.RegistrationDate,
-		u.ContractNumber,
-		u.BirthDate,
-		u.GitHubAccount,
-		u.Photo,
-		u.PhoneNumber,
-		u.ExileDate,
-		u.CityId as id,
-		ur.RoleId as id
-	FROM dbo.[User] u 
-	inner join dbo.User_Role ur on ur.UserId = u.Id
-	WHERE u.Email = @Email
+		 U.Id
+		,U.FirstName
+		,U.LastName
+		,U.Patronymic
+		,U.Email
+		,U.Password
+		,U.Username
+		,U.IsDeleted
+		,U.RegistrationDate
+		,U.ContractNumber
+		,U.BirthDate
+		,U.GitHubAccount
+		,U.Photo
+		,U.PhoneNumber
+		,U.ExileDate
+		,U.CityId as id
+		,UR.RoleId as id
```

Comment:
please revert your changes
I'm going to share one more doc. from our partners, and it contains such words:
![image](https://user-images.githubusercontent.com/18329725/126903480-4e0d4679-8197-4e67-af49-857f5caa6b24.png)